doc/release_notes_0.6.rst

```diff
@@ -58,10 +58,196 @@ for ACRN real time support will continue.
+
+   **Workaround:** None. The issues will be fixed in the next release.
+
+:acrn-issue:`1991` - Input is useless in UART Console for corner case
+   Input is useless in UART Console for a corner case,
+   demonstrated with these steps:
+
+   1) Boot to SOS
+
+   2) ssh into the SOS.
+
+   3) use "./launch_UOS.sh" to boot UOS.
+
+   4) On the host, use "minicom -s dev/ttyUSB0".
+
+   5) Use "sos_console 0" to launch SOS.
+
+   **Impact:** Failed to use UART for input in corner case.
+
+   **Workaround:** Enter other keys before typing :kbd:`Enter`.
+
+:acrn-issue:`1996` - There is an error log when using "acrnd&" to boot UOS
+   An error log is printed when starting acrnd as a background job
+   (``acrnd&``) to boot UOS. The UOS still boots up
+   normally, but prints: “Failed to open the socket(sos-lcs) to query the reason for the wake-up.
+   Activating all vms when acrnd & to boot uos."
+
+   **Impact:** UOS boots normally, but prints an error log message.
+
+   **Workaround:** None.
+
+:acrn-issue:`2267` - [APLUP2][LaaG]LaaG can't detect 4k monitor 
+   After launching UOS on APL UP2 , 4k monitor cannot be detected.
+
+   **Impact:** UOS hasn't display with 4k monitor.
+
+   **Workaround:** None.
+
+:acrn-issue:`2276` - OVMF failed to launch UOS on UP2.
+   UP2 failed to launch UOS using OVMF as virtual bootloader with acrn-dm. 
+
+   **Impact:** UOS cannot boot up using OVMF
+
+   **Workaround:** Use VSBL as virtual bootloader
+
+:acrn-issue:`2278` - [KBLNUC] Cx/Px is not supported on KBLNUC
+   C states/P states is not supported on KBL NUC.
+
+   **Impact:** Power Management states related operations cannot be using in SOS/UOS on KBLNUC
+
+   **Workaround:** None 
+
+:acrn-issue:`2279` - [APLNUC]After exiting UOS with mediator Usb_KeyBoard and Mouse, SOS cannot use the 
+   Usb_KeyBoard and Mouse
+   After exiting UOS with mediator Usb_KeyBoard and Mouse, SOS cannot use the Usb_KeyBoard and Mouse.
+   Reproduce Steps as below:
+
+   1) Insert USB keyboard and mouse in standard A port（USB3.0 port）
+
+   2) Boot UOS by sharing the USB keyboard and mouse in cmd line:
+   -s n,xhci,1-1:1-2:1-3:1-4:2-1:2-2:2-3:2-4 \
+
+   3) UOS access USB keyboard and mouse.
+
+   4) Exit UOS.
+
+   5) SOS access USB keyboard and mouse. 
+
+   **Impact:** SOS cannot use USB keyboard and mouse in such case.
+
+   **Workaround:** Unplug and plug-in the USB keyboard and mouse after exiting UOS.
+
+:acrn-issue:`2338` - [UP2]Lost 2G memory in SOS when using SBL as bootloader on UP2.
+       After using SBL as bootloader to boot UP2, SOS only has 2G memory with 4G physical memory bank.
+
+   **Impact:** lost 2G memory in SOS.
+
+   **Workaround:** Modify SBL Memory setting file "CfgData_Ext_Up2.dlt", see details in issue comments.
+
+:acrn-issue:`2522` - [NUC7i7BNH]After starting ias in SOS, there is no display 
+       On NUC7i7BNH, after starting ias in SOS, there is no display. And this issue only occurs on this model NUC which with TPC to VGA connector. 
```

Comment:
What is a TPC connector?  Do you mean DisplayPort to VGA?

Reply:
Type-C